docs/_sass/theme/uikit.scss

```diff
@@ -0,0 +1,95 @@
+// Import UIkit components
+
+// Base
+@import "../uikit/components/variables.scss";
+@import "../uikit/components/mixin.scss";
+@import "../uikit/components/base.scss";
+
+// Elements
+@import "../uikit/components/link.scss";
+@import "../uikit/components/heading.scss";
+@import "../uikit/components/divider.scss";
+@import "../uikit/components/list.scss";
+@import "../uikit/components/description-list.scss";
+//@import "../uikit/components/table.scss";
```

Comment:
Are the commented out lines serving a purpose? Can we either remove them or add a comment explaining why they are here?

Reply:
I added comment explaining why they are commented out, there is an explanation in the readme as well

Reply:
I also structured them a bit better